### DISCUSSION TITLE
MH: Rise Game Crash when using Upscaler + UpscalerBasePlugin

### DISCUSSION THREAD

**Discussion by KrisCris** [praydog/REFramework#1020] [General Questions] [answered]
I honestly don't know how to get the upscaler working. Does this only work for VR mode?

Without UpscalerBasePlugin there is no crash, but the upscaler is complaining missing backend.

I was using: 

Reframework: https://github.com/praydog/REFramework/commit/94c8eedee6106557a416f76f7e0fc77282458042
UpscalerBasePlugin: https://www.nexusmods.com/site/mods/502 v1.1.2
FSR2 DLL: https://github.com/PureDark/FidelityFX-FSR2/releases/download/2.2.1/fsr2.2.1.zip

Log:
[re2_framework_log.txt](https://github.com/praydog/REFramework/files/14840806/re2_framework_log.txt)

FYI: I am running this on my Steam Deck (SteamOS).

**Comment by praydog** [comment, marked as answer]
It is not supported in MHRise.

**Reply by KrisCris** [reply]
Alright thanks for clarify that
